src/guide/build-and-install.md

```diff
@@ -72,168 +76,179 @@ rustc 1.60.0 (7737e0b5c 2022-04-04)
 ```
 
 If you have higher versions, you're fine. If you have lower versions, you
-can run
+can run the following command to update it:
 
 ```bash
 rustup toolchain update stable
 ```
 
-to update.
+---
 
 If you get lower version numbers **and** you updated the toolchain and it
-didn't work… let's just say it's a common problem, but it doesn't have a
+didn't work... let's just say it's a common problem, but it doesn't have a
```

Comment:
Revert please. Markdown renders ellipsis and three full stops as distinct, and this should be an ellipsis.